<!-- Launch file for accuracy measurements -->

<launch>
    <group ns="saf">



        <arg name="speed_cartesian" default="20.0"/>
        <arg name="speed_jaw" default="20.0"/>

        <node name="accuracy_meas" pkg="irob_subtask_logic"
            type="accuracy_meas" output="screen">

                <param name="speed_cartesian" type="double" value="$(arg speed_cartesian)" />
                <param name="speed_jaw" type="double" value="$(arg speed_jaw)" />

                <rosparam param="arm_names">["arm_1"]</rosparam>

    </node>
    </group>
</launch>
